Source/ACE.Server/Entity/Fellowship.cs

```diff
@@ -565,6 +565,8 @@ public void SplitXp(ulong amount, XpType xpType, ShareType shareType, Player pla
         /// <param name="player">The fellowship member who originated the luminance</param>
         public void SplitLuminance(ulong amount, XpType xpType, ShareType shareType, Player player)
         {
+            return; //Hard disable luminance
```

Comment:
There's several of these `Hard disable luminance` hacks in this PR which obviously break desired behavior.